ui/src/css/extensions/custom.css

```diff
@@ -278,10 +278,6 @@
   text-decoration: none;
 }
 
-.edit-this-page:hover .pencil {
```

Comment:
@JameStark What did you delete this style? Don't you need it for the change you made in the hbs file?

Reply:
@stoobie The hover state was breaking the layout and I felt it didn't add much to the user so removed the hover state on the image so just the normal text highlight indicates hover. 

This was also in the interest of getting the feature ready to push even if we later iterate on the image hover afterwards.